examples/integration_test.py

```diff
@@ -258,6 +258,18 @@ def verify_consumer():
             print('max_msgcnt %d reached' % msgcnt)
             break
 
+    # Get current assignment
+    assignment = c.assignment()
+
+    # Get cached watermark offsets
+    # Since we're not making use of statistics the low offset is not known so ignore it.
```

Comment:
I get that we don't have stats on, but does't that mean we can at least validate that it is invalid? We don't seem to validate the high offset either?